### PR TITLE
Tests for stages 016 and 055.

### PR DESCRIPTION
**Pull request by Evildoor**
These tests were originally implemented as examples for writing real tests, but I believe that they can also be merged into master and used as normal tests. Before doing so, the following has to be done:
- [x] Conclusion of #255 and adaptation of the tests to it, and probably similar procedure for some other changes.
- [x] Update of comments regarding the pre-defined dictionaries for physical categories in 016. It was discussed on one of the meetings and decided that in this regard it is correct to repeat the definitions twice, because tests and the code must be separate entities. If the tests will import the dictionaries from the code, then they'll be unable to detect a possible incorrect change to the dictionary.
- [x] Removal of mentions that the tests are merely "samples" and "examples", as they will become regular tests.

***

To do to address comments and some other problems:
- [x] Remove pyDKB-related commits.
- [x] Remove mention of pyDKB from a1acada (former 98bbe0c).
- [x] Consider `055/test/test_documents2ttl.py`'s name.
- [x] Take a look at eb585bc and split it if necessary.
- [x] Reassess the two versions of the same test for 055.
- [x] Consider using raw strings in 055 tests.
- [x] Answer a comment about single quote test.

### CODE REVIEW

**Comment by Evildoor** [comment]
Minor point:

> I'd rather expect `test_StageName.py` to contain tests for the stage main functionality, like process() and/or main() functions (not sure there's somethin to be tested for main(), though).

Script `test_StageName.py` will not be present if stage name and stage script name are different (for example, 009: `oracleConnector` and `Oracle2JSON.py`). Did you mean `test_ScriptName.py`?

File: Utils/Dataflow/055_documents2TTL/tests/tests.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        fixed_s = "\\\\n"
+        self.assertEqual(documents2ttl.fix_string(s), fixed_s)
+
+    def test_fix_string_single_quote(self):
+        s = "'"
+        fixed_s = "\\\\'"
+        self.assertEqual(documents2ttl.fix_string(s), fixed_s)
+
+    def test_fix_string_backslash_single_quote(self):
+        s = "\'"
```
[the remark this answers is not in the document]
Comment (Evildoor) [reply Utils/Dataflow/055_documents2TTL/tests/tests.py]: Hm, looks like you are correct. Removed the test.

File: Utils/Dataflow/055_documents2TTL/tests/tests.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        iri = "<%s/document/%s>" % (documents2ttl.GRAPH, doc_id)
+        self.assertEqual(documents2ttl.get_document_iri(doc_id), iri)
+
+    def test_documents_links_manual(self):
+        data = {"dkbID": "0", "supporting_notes": [{"dkbID": "1"},
+                                                   {"dkbID": "2"}]}
+        result = "<http://nosql.tpu.ru:8890/DAV/ATLAS/document/0> "\
+                 "<http://nosql.tpu.ru/ontology/ATLAS#isBasedOn> "\
+                 "<http://nosql.tpu.ru:8890/DAV/ATLAS/document/1> .\n"\
+                 "<http://nosql.tpu.ru:8890/DAV/ATLAS/document/0> "\
```
[the remark this answers is not in the document]
Comment (Evildoor) [reply Utils/Dataflow/055_documents2TTL/tests/tests.py]: This test, in combination with the next one, was made as an illustration of possibilities in test making. It does everything as explicitly as possible, without even thinking about points such as "these variables already exist".
After some consideration, I think that this test should be kept in a separate branch as a demonstration (as you said in email), so it should be temporary removed to merge the branch into master without it, and then restored in this branch. Will do so next week, unless objections will be voiced.

File: Utils/Dataflow/055_documents2TTL/tests/tests.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                                                    iri)
+        self.assertEqual(documents2ttl.documents_links(data), result)
+
+    def test_fix_string_wrong_type(self):
+        s = 1
+        self.assertEqual(documents2ttl.fix_string(s), s)
+
+    def test_fix_string_backslash_n(self):
+        s = "\n"
+        fixed_s = "\\\\n"
```
[the remark this answers is not in the document]
Comment (Evildoor) [reply Utils/Dataflow/055_documents2TTL/tests/tests.py]: Done.

File: Utils/Dataflow/055_documents2TTL/tests/tests.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mgolosova) [reply Utils/Dataflow/055_documents2TTL/tests/tests.py]: Ok, sounds like the way to go for me, so go ahead :)

**Comment by mgolosova** [comment]
@Evildoor:
> @mgolosova:
>> I'd rather expect `test_StageName.py` to contain tests for the stage main functionality, like `process()` and/or `main()` functions.
>
> Script `test_StageName.py` will not be present if stage name and stage script name are different (for example, 009: `oracleConnector` and `Oracle2JSON.py`). Did you mean `test_ScriptName.py`?

Yes, this will work as well, why not.
Though I personally see nothing wrong in adding `test_StageName.py` or `test_main.py` or whatever even when `StageName.py` (`main.py`) does not exists... I mean, in essence it looks just as wierd as when we have main script\`s name different from the stage name (and it is not some "placeholder" name like `stage.py` or `main.py`). But, unfortunately, somehow it happened and still exists here and there... so at this point nothing will make things look even worse for me ;)

File: Utils/Dataflow/055_documents2TTL/tests/test_documents2ttl_iri_links.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+"""
+Various tests for Stage 055.
```
Comment (mgolosova) [inline Utils/Dataflow/055_documents2TTL/tests/test_documents2ttl_iri_links.py]:
```suggestion
Tests for Stage 055's functions: document_iri(), documents_links().
```

Comment (mgolosova) [reply Utils/Dataflow/055_documents2TTL/tests/test_documents2ttl_iri_links.py]: "Various" sounds a bit too uncertain )

**Comment by mgolosova** [comment]
@Evildoor, looks fine now, so I am ready to approve and merge the PR after you do what you are going to do with `test_documents_links_{manual|constructed}` finctions.
Thank you!

File: Utils/Dataflow/055_documents2TTL/tests/test_documents2ttl_iri_links.py (hunk 4 above)
Comment (Evildoor) [reply Utils/Dataflow/055_documents2TTL/tests/test_documents2ttl_iri_links.py]: Done.

File: Utils/Dataflow/055_documents2TTL/tests/tests.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Evildoor) [reply Utils/Dataflow/055_documents2TTL/tests/tests.py]: Done.

**Comment by Evildoor** [comment]
> I mean, in essence it looks just as wierd as when we have main script`s name different from the stage name (and it is not some "placeholder" name like stage.py or main.py). But, unfortunately, somehow it happened and still exists here and there... so at this point nothing will make things look even worse for me ;)

Yes, the thought about discrepancy between stage names and main scripts in some stages had occurred to me as well, but that's outside of the scope of this PR.

---

The last changes were made, so you can merge the PR - just in case, the branch should not be deleted.

**Pull request merged**